### PR TITLE
enh(release): Add Centreon Web 20.10.9 RN

### PR DESCRIPTION
**Pull request by lpinsivy**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -19,6 +19,31 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.9
+
+#### Bugfixes
+
+- [APIv2] Use poller's page ACL rights on Topology API endpoints
+- [Configuration] InfluxDB configuration columns are deleted in Broker from
```
Comment (mdebauchez) [inline en/releases/centreon-core.md:27]: I think it's 'form' and not 'from'

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -19,6 +19,31 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.9
+
+#### Correctifs
+
+- [APIv2] Use poller's page ACL rights on Topology API endpoints
+- [Configuration] InfluxDB configuration columns are deleted in Broker from
```
Comment (mdebauchez) [inline fr/releases/centreon-core.md:27]: I think it's 'form' and not 'from'

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -19,6 +19,31 @@ If you have feature requests or want to report a bug, please go to our
 
 ## Centreon Web
 
+### 20.10.9
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:23]:
```suggestion

`June 30, 2021`

```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -19,6 +19,31 @@ commerciales, vous pouvez vous rendre sur notre
 
 ## Centreon Web
 
+### 20.10.9
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:23]:
```suggestion

`30 juin 2021`

```

**Review by tuntoja — CHANGES_REQUESTED** [review]

**Review by tuntoja — APPROVED** [review]

**Pull request merged**